react/features/invite/components/dial-in-summary/NumbersList.web.js

```diff
@@ -41,76 +41,144 @@ class NumbersList extends Component<Props> {
-                    return this._renderNumberDiv(number);
-                }
+        let hrKey = 0;
+        const rows = [ <hr key = { hrKey++ } /> ];
 
-                return null;
-            });
+        Object.keys(numbers).forEach((countryName: string) => {
+            const numbersArray = numbers[countryName];
 
             rows.push(
-                <tr key = { country }>
-                    <td>{ country }</td>
-                    <td className = 'dial-in-numbers'>{ formattedNumbers }</td>
-                </tr>
+                <div
+                    className = 'number-group'
+                    key = { countryName }>
+                    {this._renderFlag(numbersArray[0].countryCode)}
+                    <div className = 'numbers-and-country' >
+                        <div className = 'country' > { countryName } </div>
+                        <ul className = 'numbers-list'>
+                            {this._renderNumbersList(numbersArray)}
+                        </ul>
+                        <ul className = 'toll-free-list' >
+                            {this._renderNumbersTollFreeList(numbersArray)}
+                        </ul>
+                    </div>
+                </div>
             );
-        }
+            rows.push(<hr key = { hrKey++ } />);
+        });
 
         return rows;
     }
 
     /**
-     * Renders a table row for a phone number.
+     * Renders a div container for a phone number.
      *
-     * @param {string} number - The phone number to display.
+     * @param {string} countryCode - The phone number to display.
+     * @private
+     * @returns {ReactElement}
+     */
+    _renderFlag(countryCode) {
+        const OFFSET = 127397;
+
+        if (countryCode) {
+            // ensure country code is all caps
+            const cc = countryCode.toUpperCase();
+
+            // return the emoji flag corresponding to country_code or null
+            const countryFlag = /^[A-Z]{2}$/.test(cc)
+                ? String.fromCodePoint(...[ ...cc ]
+                    .map(c => c.charCodeAt() + OFFSET))
+                : null;
+
+            return <div className = 'flag'>{ countryFlag }</div>;
+        }
+
+        return null;
+    }
+
+    /**
+     * Renders a div container for a phone number.
+     *
+     * @param {Array} numbers - The phone number to display.
      * @private
      * @returns {ReactElement[]}
      */
-    _renderNumberRow(number) {
-        return (
-            <tr key = { number }>
-                <td className = 'dial-in-number'>
-                    { this._renderNumberLink(number) }
-                </td>
-            </tr>
-        );
+    _renderNumbersList(numbers) {
+        return numbers.map(number =>
+            (<li
+                className = 'number-row'
+                key = { number.formattedNumber }>
+                { number.formattedNumber }
+            </li>));
+    }
+
+    /**
+     * Renders a div container for a phone number.
```

Comment:
Maybe this comment is incorrect? It renders a "toll free" string if toll free.

Reply:
Will fix it.

Reply:
Fixed.